storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2226,12 +2233,12 @@ void get_id_from_mysql_table_scan(std::string name, uint64 *id, THD *thd) {
       break;
```

Comment:
upstream dd processing code, never breaked out of the loop. I guess you need a mtr_start() before break to keep it consistent?

Reply:
U forgot to address this review comment?

Reply:
I replied in previous comment.

> in our case, It should always break. so that's why I have removed mtr_commit before dd_table_close. It would be same thing adding mtr_start here and followed by mtr_commit before dd_table_close.
Also we have assert before dd_table_close(in case it doesn't break)